app/src/main/java/fr/neamar/kiss/forwarder/Favorites.java

```diff
@@ -134,6 +135,14 @@ void onFavoriteChange() {
         favCount = favoritesPojo.size();
         LayoutInflater layoutInflater = null;
 
+        int dotColor;
+        if (isExternalFavoriteBarEnabled()) {
+            dotColor = UIColors.getPrimaryColor(mainActivity);
+        }
+        else {
+            dotColor = Color.WHITE;
```

Comment:
Can't we have the default drawable color as white and also add a stroke with some other color (the stroke will be visible only if we don't use a color filter) to make it more visible?